frontend/src/features/myinfo/utils/augmentWithMyInfoPrefill.ts

```diff
@@ -0,0 +1,17 @@
+import { keyBy } from 'lodash'
+
+import { types as myInfoTypeArray } from '~shared/constants/field/myinfo'
+import { FormFieldDto, MyInfoFormField } from '~shared/types'
+
+import { isMyInfo } from './isMyInfo'
+
+const MAP_ATTR_TO_NAME = keyBy(myInfoTypeArray, 'name')
+
+export const augmentWithMyInfoPrefill = (
```

Comment:
nit: still think this name might be misleading, what about augmentWithMyInfoPreview?